MIDAS/src/hardware/drivers/ads7138-q1.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
```

Comment:
move this whole file to the lib folder. Follow the file structure in lib. you can rebase off of mvp-sensor-integration to get this folder back, I think it got deleted somehow.